corehq/apps/api/tests/user_resources.py

```diff
@@ -691,3 +691,30 @@ def test_domain_not_returned_when_flag_not_enabled(self, *args):
         bundle.request.user = self.user
         resp = UserDomainsResource().obj_get_list(bundle)
         self.assertEqual(0, len(resp))
+
+
+class TestCommCareAnalyticsUserRolesResource(APIResourceTest):
+    resource = v0_5.CommCareAnalyticsUserRolesResource
+    api_name = 'v0.5'
+
+    def test_flag_not_enabled(self):
+        self.assertTrue(self.user.domain_memberships[0].is_admin)
```

Comment:
nit: I would not recommend this check (or any input condition) as a test here as it the failure message would be unrelated to the actual test case `test_flag_not_enabled` when it fails.

Consdering this is an input condition for the test case, I think explicitly setting the value as done earlier is a better approach.
`self.user.domain_memberships[0].is_admin = True`

Reply:
@ajeety4 Are you proposing to remove the test case?

Reply:
Yes instead of using a test check `self.assertTrue(self.user.domain_memberships[0].is_admin)` , simply set the desired value  `self.user.domain_memberships[0].is_admin = True` .

Reply:
Cool. I ended up removing the checks altogether since they don't really add to the test. See https://github.com/dimagi/commcare-hq/pull/34987/commits/bcd4fe51b53279fe02d3e8c57d74d34389e2e5f9